apps/vlasov.c

```diff
@@ -290,12 +298,11 @@ gkyl_vlasov_app_new(struct gkyl_vm *vm)
   // as need pointers to colliding species' collision objects
   // allocated in the previous step
   for (int i=0; i<ns; ++i)
-    if (app->species[i].collision_id == GKYL_LBO_COLLISIONS
-      && app->species[i].lbo.num_cross_collisions)
+    if (app->species[i].collision_id == GKYL_LBO_COLLISIONS &&
+        app->species[i].lbo.num_cross_collisions)
       vm_species_lbo_cross_init(app, &app->species[i], &app->species[i].lbo);
 
-  // initialize each species source terms: this has to be done here
-  // as they may initialize a bflux updater for their source species
```

Comment:
I would like to leave this longer comment in so no future developers try to move the source terms into the species initialization unless @kltrbrds says that this comment can be safely deleted (but I think source terms can still fetch a "source species" if they are using the boundary flux from another species).

Reply:
done